app/src/main/java/com/cornellappdev/android/eatery/ui/viewmodels/HomeViewModel.kt

```diff
@@ -107,6 +107,20 @@ class HomeViewModel @Inject constructor(
         }
     }.stateIn(viewModelScope, SharingStarted.Eagerly, listOf())
 
+    /**A [StateFlow] that emits a list of all eateries sorted by nearest proximity
+     *
+     * Sorted (by descending priority): Open/Closed, Walk Time
+     * */
+    val eateriesByDistance: StateFlow<List<Eatery>> = eateryFlow.map { apiResponse ->
+        when (apiResponse) {
+            is EateryApiResponse.Error -> listOf()
+            is EateryApiResponse.Pending -> listOf()
+            is EateryApiResponse.Success -> {
+                apiResponse.data.sortedBy { it.getWalkTimes() }.sortedBy { it.isClosed() }
+            }
+        }
+    }.stateIn(viewModelScope, SharingStarted.Eagerly, listOf())
+
```

Comment:
What happened to `nearestEateries`? Are we still using it somewhere and therefore it can't be removed? Since it seems like you stopped using it the home screen composable.

Reply:
Went ahead and deleted it since eateriesByDistance replaces it.

Reply:
There's also a val tweenHeight around line 309, I'm pretty sure it was used to separate the Favorites and NearestEateries carousels before NearestEateries was removed. Not sure if it's needed anywhere else, should I delete it as well?

Reply:
Oh hm I remember adding `tweenHeight` as a very minor padding fix to the screen for favorites animation. I guess if the padding is fine now without it then you can feel free to delete!